### ISSUE TITLE
Integrate Magic Eden provider

### ISSUE THREAD

**Issue by somethingbeta** [secretkeylabs/sats-connect#184]
Hi All, 

A bit stuck with this one, I am working on a dApp supporting, the xverse, unisat and magic eden wallets. I see from the Magic Eden api its sats-connect compatible. 

On my test browser I have the extensions of all three installed. This is what i am seeing

When i run the ME example, only the ME wallet comes up. Note further the ME examples now use a deprecated api
When i run the sats-connect i only see, unisat and xverse if i run this using the "Wallet" instance. 
A call to getProviders() only returns the xverse provider. 

Been digging around quite a bit for this, couldnt find anything. 

In summary, would like to be in a state where all three wallets show up as connectable and be used uniformly. 

Possible Labels: Question, Help
